index.html

```diff
@@ -214,5 +214,70 @@ <h2>
         </div>
       </div>
     </div>
+    <div class="contact-section">
+      <p class="para-info">
+        I'm always <br />interested in hearing<br />
+        about new projects,<br />
+        so if you'd like to<br />
+        chat please get in touch.
+      </p>
+      <form action="https://formspree.io/f/{form_id}">
+        <ul class="list-container">
```

Comment:
- [ ] [REQUIRED] Good job so far! Form should use the `action` and `method` attribute to integrate with [formspree service](https://formspree.io/html), `action` value should be similar to `https://formspree.io/f/{form_id}` || {form_id} should have an `ID` provided by [formspree service](https://formspree.io/html) please sign up that and get an `ID` please || Please implement `method` value equals `POST` like this => `<form action="..." method="POST" >` 🙏🏼